src/components/RadioButton/RadioButton.st.css

```diff
@@ -0,0 +1,181 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback, font, applyOpacity;
+}
+
+:import {
+    -st-from: "wix-ui-core/index.st.css";
+    -st-named: RadioButton;
+}
+
+:import {
+    -st-from: "../../common/palette.st.css";
+    -st-named: errorColor;
+}
+
+/*Defaults*/
+:vars {
+    DefaultTextColor: color-5;
+    DefaultBorderColor: color-5;
+    DefaultIconColor: color-8;
+    DefaultBackgroundColor:color-1;
+    DisabledColor: gray;
+}
+
+/*Overrides*/
+:vars {
+    TextColor: --overridable;
+    BorderColor: --overridable;
+    IconColor:  --overridable;
+    BackgroundColor:--overridable;
+}
+
+:vars {
+    CurrentBorderColor: color(fallback(value(BorderColor), value(DefaultBorderColor)));
+    CurrentIconColor: color(fallback(value(IconColor), value(DefaultIconColor)));
+    CurrentTextColor: color(fallback(value(TextColor), value(DefaultTextColor)));
+    CurrentBackgroundColor: color(fallback(value(BackgroundColor), value(DefaultBackgroundColor)));
+    LabelTextFont: "{theme: 'Body-M', size: '16px', lineHeight: '1.5em'}";
+}
+
+.labelWrapper {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+}
+
+.root {
+    -st-extends: RadioButton;
+    -st-states: checked, disabled, box, error;
+    display: flex;
+    line-height: 20px;
+}
+
+.root:error .checkmark {
+    border-color: color(value(errorColor));
+}
+
+.root:box:disabled {
+    cursor: default;
+}
+
+.root:box:hover, .root:box:focus-within {
+    border-color: applyOpacity(value(CurrentBorderColor), 1);
+}
+
+.root:box.disabled:hover:not(:checked), .root:box:disabled:focus-within:not(:checked) {
+    border-color: applyOpacity(value(CurrentBorderColor), 0.6);
+}
+
+.root:box:checked {
+    background-color: applyOpacity(value(CurrentIconColor), 0.1);
+    border-color: value(CurrentIconColor);
+}
+
+.root:box:checked:not(:disabled):hover, .root:box:checked:not(:disabled):focus-within {
+    border-color: applyOpacity(value(CurrentIconColor), 0.6);
+}
+
+.root:box:not(:disabled):hover .icon {
+    border-color: applyOpacity(value(CurrentBorderColor), 1);
+}
+
+.root::label {
+    width: 100%;
+    font: font(value(LabelTextFont));
+    color: applyOpacity(value(CurrentTextColor),1);
+}
+
+.root::icon {
+    display: flex;
+    position: relative;
+    margin-bottom: 1px;
+    top:3px;
+    cursor: pointer;
```

Comment:
Please move the `cursor: pointer` to the `.root` since the `pointer` should be on both the icon and the label.
In addition, be sure to add `cursor: default` on `disabled` state.